Readme.md

```diff
@@ -43,6 +43,7 @@ Want to organize a local NodeBots Day event. You should!
 ### North America
  - [Boston, Massachusetts, USA](2016/boston/)
  - [Dayton, OH, USA](2016/dayton/)
+ - [Dalas, Texas, USA](2016/dallas/)
```

Comment:
yup, that's important. at least I spelled the link right :P
I fixed the typo.

Ron

Ron Dagdag
817-458-8885

https://www.*linkedin*.com/in/*rondagdag*
<https://www.linkedin.com/in/rondagdag>



On Wed, Jun 22, 2016 at 11:18 AM, Linda Nichols <notifications@github.com>
wrote:

> In Readme.md
> <https://github.com/nodebots/nodebotsday/pull/223#discussion_r68085189>:
>
> > @@ -43,6 +43,7 @@ Want to organize a local NodeBots Day event. You should!
> >  ### North America
> >   - [Boston, Massachusetts, USA](2016/boston/)
> >   - [Dayton, OH, USA](2016/dayton/)
> > + - [Dalas, Texas, USA](2016/dallas/)
>
> Looks like you have a small typo here 😉
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodebots/nodebotsday/pull/223/files/cec01ec946cda5cf00d9aae5ea41e3f236f6707e#r68085189>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADsxhmSpQAZfnJJSREb3JE49xrBwq-Biks5qOWBigaJpZM4I79LO>
> .
>